spate_190313.R

```diff
@@ -150,6 +150,8 @@ spate_model<-function(x, varnames)
     (1-failure_due_to_poor_implementation)
   
   # Cost 11: cost of lossing the current pasture production by shifting to crop production
+  #CORY NOTE ####
+  #Would all pasture be lost? Is 'current_value_pasture' the whole pasture area of the whole dam?
```

Comment:
Hi Cory,
Thanks for the change. 
This is the change from pasture area into crop area. Since it is changed, the loss in pasture is a considered totally. However, the total farming area is not expected to be under currently under grazing area. That's why I consider the proportion of area that is currently under pasture that changed into crop (i.e. prop_area_converted_crop), which is written in the code below the above mentioned codes. I put it below for explanation.
value_pasture_lost<-total_farming_area*prop_area_converted_crop*value_pasture_without_interv